.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: Build manage-help-content-publisher
+
+on:
+  pull_request:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
```

Comment:
does it automatically build other branches in a separate build?

Reply:
This tells it to only run a build when a PR (or a draft) is raised, when the workflow is manually triggered, or when there's a push to main.

Reply:
got it, makes sense, I assume when a PR is raised or a new push to a PR